README.md

```diff
@@ -307,6 +307,29 @@ You can customize additional settings in your `.env` file:
+| `REDIS_SENTINEL_MASTER_NAME` | Name of the Redis master node in Sentinel configuration.                             | `"master"`               | Conditional (required if using Sentinel)                                                    |
+| **Redis**                    |                                                                                      |                          |                                                                                             |
+| `REDIS_HOST`                 | Redis server hostname or IP address for Redis configuration.                    | `""`                     | Conditional (required for standalone Redis)                                                 |
+| `REDIS_PORT`                 | Redis server port for Redis configuration.                                      | `6379`                   | Conditional (required for standalone Redis)                                                 |
+| **Both**                     |                                                                                      |                          |                                                                                             |
+| `REDIS_DB`                   | Redis database number to use for caching.                                            | `0` (Sentinel) / `0` (Standalone) | Optional                                                                                    |
+| `REDIS_MAX_CONNECTIONS`      | Maximum number of connections in the Redis connection pool.                          | `10`                     | Optional                                                                                    |
+| `REDIS_RETRY_TIMEOUT`        | Enable retry on timeout for Redis operations.                                        | `true`                   | Optional                                                                                    |
+| `REDIS_DECODE_RESPONSES`     | Automatically decode Redis responses to strings.                                     | `true`                   | Optional                                                                                    |
+| `REDIS_CLIENT_NAME`          | Client name identifier for Redis connections.                                        | `"stac-fastapi-app"`     | Optional                                                                                    |
+| `REDIS_HEALTH_CHECK_INTERVAL`| Interval in seconds for Redis health checks.                                         | `30`                     | Optional                                                                                    |
+
+> [!NOTE]
+> Use either the Sentinel configuration (`REDIS_SENTINEL_HOSTS`, `REDIS_SENTINEL_PORTS`, `REDIS_SENTINEL_MASTER_NAME`) OR the Redis configuration (`REDIS_HOST`, `REDIS_PORT`), but not both.
+
```

Comment:
We should add a REDIS_ENABLED global env var to turn on/off redis. The try/except blocks can add latency to the application if someone doesn't want to use Redis and the api keeps trying to connect.